projects/components/src/select/select.component.ts

```diff
@@ -314,6 +314,7 @@ export class SelectComponent<V> implements ControlValueAccessor, AfterContentIni
   private setSelection(value?: V): void {
     this.selected = value;
     this.selected$ = this.buildObservableOfSelected();
+    this.cdr.markForCheck();
```

Comment:
Since the issue is coming when using select with forms, please add this method inside the `writeValue` method. `setSelection` is being called for non form situations as well